dna-src/zomes/my_zome/code/src/lib.rs

```diff
@@ -193,18 +217,8 @@ mod my_zome {
 
     #[zome_fn("hc_public")]
     fn create_edge(edge: Edge) -> ZomeApiResult<GetResponse<Edge>> {
-        let app_entry = Entry::App("edge".into(), edge.clone().into());
-        let _ = hdk::commit_entry(&app_entry)?;
-
-        // link each new goal to the anchor
-        let anchor_address = Entry::App(
-            "anchor".into(), // app entry type
-            "edges".into() // app entry value
-        ).address();
-
-        hdk::link_entries(&anchor_address, &app_entry.address(),  "anchor->edge", "")?;
-
-        Ok(GetResponse{entry: edge, address: app_entry.address()})
+        let entry_address = inner_create_edge(&edge).unwrap();
```

Comment:
Using the following is preferable here because the ? will expand this into a `match` statement and cause the function to return an `Err` result, whereas calling `.unwrap()` can cause the code to `panic` if a bad input was given

```suggestion
        let entry_address = inner_create_edge(&edge)?;
```